README.md

```diff
@@ -1 +1,11 @@
-# eq-concourse-base-images
\ No newline at end of file
+# eQ Infrastructure Build Image
+
+
+This repo holds the Dockerfile used to build the infrastructure base image.
+
+
+## Docker
+The Dockerfile can be built using:
+```bash
+docker build -t eq-infrastructure-build-image:latest .
```

Comment:
We should be using `make build` as the build command in order to maintain consistency with other eq projects.

Reply:
I don't think it's worth adding a Makefile just for this one command, especially when it is primary not used for local dev.